routes/api/v1/favorites.js

```diff
@@ -6,6 +6,47 @@ const configuration = require('../../../knexfile')[environment];
 const database = require('knex')(configuration);
 const fetch = require("node-fetch");
 const env = require('dotenv').config()
+const CurrentForecast = require('../../../current_forecast')
+
+async function fetchCoordinates(location) {
+  let coordinates = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${location},co&key=${process.env.GOOGLE_KEY}`)
+    .then(response => response.json())
+    .then(result => result.results[0].geometry.location)
+    .then(coordinate_data => `${coordinate_data.lat},${coordinate_data.lng}`)
```

Comment:
I think one of the purposes of the async/await syntax is to avoid chaining `.then` over and over. You could refactor this to something like below (but it would do exactly the same thing, I think it's just personal preference for readability)

```
let json = await coordinates.json();
let coordinate_data = await json.results[0].geometry.location;
let coordinates = await `${coordinate_data.lat},${coordinate_data.lng}`;
```